src/lib/sectionList/listViews/sectionListViewsConfig.ts

```diff
@@ -59,7 +59,6 @@ export const defaultModelViewConfig = {
             'createdBy',
             'href',
             'id',
-            'lastUpdatedBy',
```

Comment:
Why is this removed? This is default and show be available for all models. If not, it can be overridden for a specific model with `overrideDefaultAvailable`.